File: Java/program/Shirt.java
public class Shirt extends Clothing{
    // Private attributes
    private String color;
    private String sleeve_type;

    // Konstruktor untuk memberikan nilai kosong pada attribute
    public Shirt(){
        this.color = "";
        this.sleeve_type = "";
    }

    // Konstuktor dengan parameter untuk memberikan nilai sesuai masukan pada atribute
    public Shirt(String color, String sleeve_type){
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    // Setter

    public void set_color(String color){
        this.color = color;
    }

    public void set_sleeve_type(String sleeve_type){
        this.sleeve_type = sleeve_type;
    }

    // Getter

    public String get_color(){
        return this.color;
    }

    public String get_sleeve_type(){
        return this.sleeve_type;
    }
}

File: CPP/program/Shirt.cpp
// Import library dari induk
#include "Clothing.cpp"

// Class Shirt, yang merupakan anak dari Class Clothing
class Shirt : public Clothing
{
    // Atribut private yang dimiliki shirt
    private:
        string color, sleevetype;

    // Method yang bisa dilakukan oleh class Shirt
    public:
        Shirt() {
            this->color = "";
            this->sleevetype = "";
        }
        Shirt(string color, string sleevetype, string gender) {
            this->color = color;
            this->sleevetype = sleevetype;
        }

        // Get color 
        string get_color() {
            return this->color;
        }

        // Set color
        void set_color(string color) {
            this->color = color;
        }

        // Get sleevetype 
        string get_sleevetype() {
            return this->sleevetype;
        }

        // Set sleevetype
        void set_sleevetype(string sleevetype) {
            this->sleevetype = sleevetype;
        }

        ~Shirt(){

        }
};